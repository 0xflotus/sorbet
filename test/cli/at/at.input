test/cli/at/at.rb
