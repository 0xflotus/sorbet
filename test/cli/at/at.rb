# @typed
1 + "s"
