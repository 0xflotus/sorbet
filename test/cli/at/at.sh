#!/bin/bash
main/sorbet @test/cli/at/at.input 2>&1

main/sorbet @does_not_exist 2>&1 || true
