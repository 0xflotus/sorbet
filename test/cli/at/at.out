test/cli/at/at.rb:2: `String("s")` doesn't match `Integer` for argument `arg0` http://go/e/7002
     2 |1 + "s"
        ^^^^^^^
    https://git.corp.stripe.com/stripe-internal/sorbet/tree/master/rbi/core/integer.rbi#L72: Method `Integer#+` has specified `arg0` as `Integer`
    72 |      arg0: Integer,
              ^^^^^
  Got String("s") originating from:
    test/cli/at/at.rb:2:
     2 |1 + "s"
            ^^^
Errors: 1
